lib/contentful/Articles.js

```diff
@@ -29,7 +29,7 @@ export default class ContentfulArticles extends ContentfulApi {
 
     const response = await this.callContentful(query);
 
-    const articles = response.data.articleCollection.items
+    const articles = response.data?.articleCollection.items
```

Comment:
I can split this one out to a new PR if you like, I just added it so that it doesn't explode when I don't have contentful credentials locally